tracker.go

```diff
@@ -139,31 +138,18 @@ func (c *Client) CreateTrackerList(opts ...*CreateTrackerOptions) error {
 	// }
 	// The keys inside of the 'trackers' map (0, 1 in the example) get discarded
 	// by the API endpoint, so are not important.
-	trackers := make(map[string]map[string]interface{}, len(opts))
-	for i, key := 0, 0; i < len(opts); i++ {
-		if m := opts[i].toMap(); m != nil {
-			trackers[strconv.Itoa(key)] = opts[i].toMap()
-			key++
-		}
-	}
-	req := map[string]interface{}{"trackers": trackers}
-	return c.post(nil, "trackers/create_list", req, nil)
+	req := map[string]interface{}{"trackers": param}
+	//this endpoint does not return a response so we return true here
```

Comment:
Let's keep really clean comments by always capitalizing the start of the phrase and giving spaces between the start of `//` and the comment itself.